xla/service/gpu/transforms/command_buffer_scheduling_test.cc

```diff
@@ -1117,79 +1117,6 @@ TEST_F(CommandBufferSchedulingTest, AsyncAlltoAll) {
                             });
 }
 
-TEST_F(CommandBufferSchedulingTest, DynamicSliceFusionDynamicSlicing) {
```

Comment:
Can we turn this test into a useful one for the new behavior?

E.g., maybe we can check that we have command buffers in certain cases, now?

Reply:
I have not added a test for that, because that is not supported right now. This patch is getting bigger, I will add that support in a separate patch later. For now, I have ensured that command buffer doesn't consume dynamic slice fusion when there are dynamic addresses in it.